v3/src/components/component-title-bar.tsx

```diff
@@ -30,16 +31,36 @@ export const ComponentTitleBar = observer(function ComponentTitleBar(
     }
   }
```

Comment:
Shouldn't this call to `tile.setTitle` be wrapped in `applyModelChange`?

Reply:
Good call 👍